File: docs/code-examples/text_log.cpp
// Log a `TextLog`

#include <rerun.hpp>

#include <cmath>

int main() {
    auto rec = rerun::RecordingStream("rerun_example_text_log");
    rec.spawn().throw_on_failure();

    rec.log("log", rerun::TextLog("Application started.").with_level(rerun::TextLogLevel::INFO));
}

File: docs/code-examples/text_log.rs
//! Log a `TextLog`

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_text_log").spawn()?;

    rec.log(
        "log",
        &rerun::TextLog::new("Application started.").with_level("INFO"),
    )?;

    Ok(())
}
